src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3647,6 +3629,62 @@ def add_vnet_integration(cmd, name, resource_group_name, vnet, subnet, slot=None
     return mod_vnet
 
 
+def _validate_subnet(cli_ctx, subnet, vnet, resource_group_name):
```

Comment:
This method returns different resource types. It can be a subnet or a vnet. Be careful.

Reply:
Hi @qwordy As far as I can see, it always return a subnet resource id. Either if the input subnet id is valid or constructed from the vnet and subnet input. Which line do you see a potential vnet returned?

Reply:
You are right. "subnets" is child type. My mistake.